### PR TITLE
BBPP134-1942 adding back ab and 2/3 special case  regex

### PR DESCRIPTION
**Pull request by jdcourcol**
as discussed in https://bbpteam.epfl.ch/project/issues/browse/BBPP134-1942

### CODE REVIEW

**Review by arnaudon — APPROVED** [review]

File: atlas_placement_hints/app/metadata/isocortex_metadata.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     },
     "layers": {
         "names": ["layer_1", "layer_2", "layer_3", "layer_4", "layer_5", "layer_6"],
-        "queries": ["@^(?!SSp-bfd-[A-Z][0-9]$).*1$", "@^(?!SSp-bfd-[A-Z][0-9]$).*2$", "@^(?!SSp-bfd-[A-Z][0-9]$).*3$", "@^(?!SSp-bfd-[A-Z][0-9]$).*4$", "@^(?!SSp-bfd-[A-Z][0-9]$).*5$", "@^(?!SSp-bfd-[A-Z][0-9]$).*6[ab]$"],
+        "queries": ["@^(?!SSp-bfd-[A-Z][0-9]$).*1[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*2[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*[^/]3[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*4[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*5[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*6[ab]$"],
```
Comment (mgeplf) [inline atlas_placement_hints/app/metadata/isocortex_metadata.json:11]: AFAIK, only the `[ab]?` is needed for layer 6.

Looking at the hierarchy, there is some strangeness in layer 2, which surprises me:
```
"acronym": "ENTl2a",
"acronym": "ENTm2a",
"acronym": "ENTl2b",
"acronym": "ENTm2b",
```
However, this isn't in `SSp-bfd`, so I don't think we need the `[ab]` anywhere but layer 6.

**Comment by lecriste** [comment]
Shall we add a test as @arnaudon mentioned [here](https://bbpteam.epfl.ch/project/issues/browse/BBPP134-1942?focusedId=248719&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-248719)?

**Comment by jdcourcol** [comment]
I am not sure if the last "?" is needed or not.

**Comment by mgeplf** [comment]
> I am not sure if the last "?" is needed or not.

Do you mean in `"@^(?!SSp-bfd-[A-Z][0-9]$).*6[ab]?$"]`?
In the current hierarchy, there is only `6a` and `6b`; so technically not needed.
I'm sure it will bite us in the future not having it though :(

**Pull request merged**